CMakeLists.txt

```diff
@@ -23,11 +23,19 @@
 cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
 
 # Install prefix
-set(CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE PATH "Install path prefix, prepended onto install directories")
-
-#Adding CMAKE_PREFIX_PATH
-list( APPEND CMAKE_PREFIX_PATH /opt/rocm/llvm /opt/rocm )
+if (WIN32)
+  SET( CMAKE_INSTALL_PREFIX "C:/hipSDK" CACHE PATH "Install path" FORCE )
```

Comment:
@TorreZuk when do we change this into something more appropriate for a windows install path?

Reply:
> @TorreZuk when do we change this into something more appropriate for a windows install path?

@saadrahim I thought you were organizing the packaging people.  But didn't you mention last week someone was assigned now?   After our next meeting if there is clarity it can be tasked up then.  For now the libs I expect are installing into C:/hipSDK.
An SDK installer builder can likely make a package/msi from contents in hipSDK and put them in a new tree, and merge with C:/hip or not merge...   This is fine for now as matches the other libs more or less.   Also the symlink duplication weren't you looking into the preferred solution?

Reply:
It is slow progress with packaging. I think we should consider using an environment variable like HIP_PATH or equivalent for this in the future in our code instead of  hard coding it.

Reply:
Yes that makes sense to use a variable, wasn't some document suggesting not to use environment variables, but anyway my mistake for not going that route from the start but let's leave them all until after the release and then switch to use one and set the desired default for the installations.    The only problem with injecting into the folder along with hip is the cleanup afterwards as not sure what tool can remove the contents of a zip from a folder.   And do we keep the common bin and include as done now?   Is there a ticket for the packaging and install ?